### PR TITLE
Separate tests, update doc

### PR DESCRIPTION
**Pull request by PavelBulat**

### CODE REVIEW

File: modules/features/falcon_feature_media/falcon_feature_media.info.yml [hunk 1]
```diff
@@ -12,4 +12,4 @@ dependencies:
   - path
   - svg_image
 version: 8.x-1.0
-package: Falcon
+package: Falcon Configurations
```
Comment (PavelBulat) [inline modules/features/falcon_feature_media/falcon_feature_media.info.yml:15]: It is fix features which came after rebase.

File: Makefile [hunk 2]
```diff
@@ -105,6 +105,10 @@ install: | prepare
 	fi
 	$(call message,Congratulations! You installed $(PROJECT_NAME)!)
 
+add\:additional-modules:
```
Comment (PavelBulat) [inline Makefile]: I create new command because i do not know how i can use env variables in config CI.

Comment (duozersk) [reply Makefile]: We don't need this additional command, you can use the `make drush en \$\(ADDITIONAL_MODULES\)` command in the CI.

File: .circleci/config.yml [hunk 3]
```diff
@@ -39,12 +39,16 @@ jobs:
...
 
       - run:
-          name: Run API tests
-          command: make tests:run api
+          name: Install additional modules
+          command: make add:additional-modules
+
+      - run:
+          name: Run basic and additional groups tests
+          command: make tests:run -- -g basic -g additional
```
Comment (duozersk) [inline .circleci/config.yml:51]: Yep, good idea; we definitely need to run both groups after enabling additional modules.

File: docs/testing.rst [hunk 4]
```diff
@@ -62,6 +62,17 @@ Writing tests
...
+it will cover basic installation distribution or not.
+If it cover basic installation distribution it should added to ``basic`` tests group. If is not it that it should added to ``additional`` group.
+**For example:** ::
+
+    /**
+     * @group basic
+     */
+    public function testBasic()
+
+If you wrote test in ``additional`` group that you should add module, which test covered, to ``ADDITIONAL_MODULES`` into ``env.default`` file.
```
Comment (duozersk) [inline docs/testing.rst]: We definitely need to rewrite this...

**Review by duozersk — CHANGES_REQUESTED** [review]

File: .circleci/config.yml (hunk 3 above)
Comment (duozersk) [reply .circleci/config.yml:51]: A second thought - we can put all the basic tests into the `additional` group too - not sure what approach is better :)

Comment (PavelBulat) [reply .circleci/config.yml:51]: I think we should not add all basic tests to additional group, becouse it is Additional tests only. Probably we can create group All_tests or smth like this. (but if we will not have other groups we will can used  make tests:run )

File: docs/testing.rst (hunk 4 above)
Comment (PavelBulat) [reply docs/testing.rst]: I think it is the difficult task for me =) I afraid if i will rewrite it, it will be bad too.

**Review by duozersk — APPROVED** [review]

**Pull request merged**